website/docs/r/google_folder_organization_policy.html.markdown

```diff
@@ -96,6 +112,10 @@ The `allow` or `deny` blocks support:
 
 * `values` - (Optional) The policy can define specific values that are allowed or denied.
 
+The `restore_policy` block supports:
+
+* `default` - (Required) If true, then the default Policy is restored. If false, then any configuration is acceptable.
```

Comment:
Actually, if it's set to false then it throws an error.

Reply:
changed